cylc/flow/scripts/lint.py

```diff
@@ -501,23 +499,36 @@ def main(parser: COP, options: 'Values', *targets) -> None:
             checks = parse_checks(check_names)
             for file_ in get_cylc_files(target):
                 LOG.debug(f'Checking {file_}')
-                count += check_cylc_file(file_, checks, options.inplace)
+                count += check_cylc_file(
+                    target,
+                    file_,
+                    checks,
+                    options.inplace,
+                )
 
         # Summing up:
-        if count > 0:
-            color = Fore.YELLOW
-        else:
-            color = Fore.GREEN
-
         if options.linter == 'all':
             checks_done = "728 & style"
         else:
             checks_done = options.linter
-        msg = (
-            f'Checked {target} against {checks_done} '
-            f'rules and found {count} issues.'
-        )
-        print(f'{color}{"-" * len(msg)}\n{msg}')
+
+        if count > 0:
+            msg = (
+                f'\n{Fore.YELLOW}'
```

Comment:
Only print the newline if there was something to report, avoids a blank line for a workflow with no lint.